docs/tools/vale/chef/acronyms.yml

```diff
@@ -0,0 +1,24 @@
+extends: conditional
+message: "'%s' has no definition."
+link: https://docs.microsoft.com/en-us/style-guide/acronyms
+level: suggestion
+ignorecase: false
+# Ensures that the existence of 'first' implies the existence of 'second'.
+first: '\b([A-Z]{3,5})\b'
+second: '(?:\b[A-Z][a-z]+ )+\(([A-Z]{3,5})\)'
+# ... with the exception of these:
+exceptions:
```

Comment:
I've found that the lints make the most sense if you copy the unique entries from this list to `vale/microsoft/acronyms.yml` and delete this linter. Otherwise you end up with false positives from competing lints.